app/schema/answer.py

```diff
@@ -58,14 +56,12 @@ def check_user_input(user_input):
 
     def get_typed_value(self, raw_input):
 
-        user_input = bleach.clean(raw_input)
```

Comment:
In reality we should probably show an error to the user if they put a `<` in their name (like non-numeric input in numeric fields). I'm thinking WTForms stuff (regex acceptance list for text input?)